src/main/java/com/ss/lms/entity/Loans.java

```diff
@@ -1,26 +1,61 @@
 package com.ss.lms.entity;
 
 import java.io.Serializable;
+
 import java.time.LocalDate;
 
+import javax.persistence.Column;
+import javax.persistence.EmbeddedId;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
 import org.springframework.stereotype.Component;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+@Entity
+@Table(name = "tbl_book_loans")
 @Component
 public class Loans implements Serializable {
 	/**
 	 * 
 	 */
 	private static final long serialVersionUID = -8268329077429523053L;
 
-	private Branch branch;
+	@EmbeddedId
+	private LoansId loansId;
 
-	private Book book;
-	private Borrower borrower;
+	@Column(name = "dateOut")
 	private LocalDate dateOut;
+
+	@Column(name = "dueDate")
 	private LocalDate dueDate;
-	private LocalDate dateIn;
 
+	@Column(name = "dateIn")
+	private LocalDate dateIn;
 	
-
+	@JsonIgnore
+	@ManyToOne(fetch = FetchType.LAZY)
+	@JoinColumn(name = "cardNo", insertable = false, updatable = false)
+	private Borrower borrower;
+	
+	
+	@JsonIgnore
+	@ManyToOne(fetch = FetchType.LAZY)
+	@JoinColumn(name = "bookId", insertable = false, updatable = false)
+	private Book book;
+	
+	@JsonIgnore
+	@ManyToOne(fetch = FetchType.LAZY)
+	@JoinColumn(name = "branchId", insertable = false, updatable = false)
```

Comment:
why is this being @jsonignored as well. Wouldnt you want this serialized and desrialized when sending info about loans